lbrynet/daemon/DaemonServer.py

```diff
@@ -54,10 +54,9 @@ def start(self, use_auth):
 
     @defer.inlineCallbacks
     def stop(self):
-        if self._api is not None:
-            yield self._api._shutdown()
-        if self.server_port is not None:
```

Comment:
I think you should keep `server_port.stopListening()`, maybe in `_shutdown` instead